docs/website/contents/for-developers/Genesis.md

```diff
@@ -0,0 +1,718 @@
+Either the logic for using Bootstrap Peers could be retained despite the introduction of the Genesis implementation, guarded behind configuration, or else bespoke Genesis snapshot config file could indirectly force the syncing node to only use Bootstrap Peers without actually being aware of their special status.
+
+## Follow-Up Work
+
+The following additional work on Genesis is suggested, as priorities and resources permit.
+
+- The HAA explicitly excludes disaster scenarios, but it is likely useful to anticipate/explore how the implementation handles disaster scenarios before they happen on mainnet, especially violations of Chain Growth.
+- As mentioned above, the engineers have always assumed Sgen = Scg.
+  Any such coincidence invites accidental conflations, so it will likely be useful to consider/excercise smaller Sgens, such as Sgen = 0.5×Scg.
+  This could result in some specifications and/or implementations being refined to invoke Scg directly instead of indirectly via Sgen, for example.
+- The Parameter Tuning section above characterizes the upper bound of unnecessary delay by assuming that disconnected peers are not replaced.
+  For example, if a leashing peer were immediately replaced, it could start with a significant number of LoP tokens and pick up where its predecessor left off.
+  A reasonable compromise would be for the Diffusion Layer to only replace peers disconnected with prejudice after a moderate delay, eg 15 minutes.
+  If the HAA is indeed satisfied, then an extra delay when replacing peers would not introduced any extra delay to the sync.
+- Refine some names, such as PreObjectors being called dissenting jumpers in the code, the Devoted BlockFetch implementation not including that name, Disengaged peers incorrectly suggesting they're inactive, etc.
+- Tighten the PreObjector MsgFindIntersect timeouts.
+  The Network team advises that this is not a high priority vector, but it'd still be preferable to simply eliminate it.
+- The LoP and DBF are tuned based on an assumption that modern empty blocks are not particularly concentrated.
+  For example, ~4% of the ~650000 most recent blocks were empty as of writing.
+  Unless a bulk of those are contiguous, the LoP and DBF buffers will likely absorb one out of, say, 15 blocks being empty.
+  It may be worth checking for a string of empty blocks.
+  It also may be necessary to consider that the disaster recovery plan could create such a string.
+- Potential test improvements:
+   - Integrate GSM into the peer simulator and test for consecutive periods of Syncing and CaughtUp states.
+     This would require some rethinking on expressivity of the point schedules.
+   - Adjust genesis tests to allow for a non-zero GDD rate limit.
+     [WIP code here](https://github.com/IntersectMBO/ouroboros-consensus/commit/4ff3138e6b6bfaf06f7f9ba49bab4d8b1e77a969).
```

Comment:
How are we going to ensure that this branch remains open? I can imagine that after a few years, if this improvement is not made, we might have to delete the branch because it has bitrot. Maybe a dead-links checker would suffice to detect this link is stale.